tests/common/usage_helper.hpp

```diff
@@ -61,16 +60,20 @@ class MockSlaveUsage {
     if (usages.isError()) {
       LOG(ERROR) << "Json Usage failed: " << usages.error() << std::endl;
     }
-    // Take first fixture only.
-    results = usages.get().resource_usage(0);
+
+    results = usages.get();
   }
 
   process::Future<ResourceUsage> usage() {
-    return results;
+    if (iteration >= results.resource_usage_size())
```

Comment:
We should do a sweep and add '{}' to all block (also one liners) per Google C++ style guide